CMakeLists.txt

```diff
@@ -27,6 +27,10 @@ set(WPE_BACKEND
     ""
     CACHE STRING "Name of the backend library to load, instead of libWPEBackend-default.so"
 )
+set(WPE_ENABLE_FS
+    ON
+    CACHE STRING "(Experimental) Enable file system path querying"
```

Comment:
Would want to be `OFF` before landing but putting as `ON` to make sure the build is successful.